setup/bindir/cloud-sysvmadm.in

```diff
@@ -23,7 +23,26 @@
 #set -x
 
 usage() {
-  printf "\nThe tool stopping/starting running system vms and domain routers \n\nUsage: %s: [-d] [-u] [-p] [-m] [-s] [-r] [-a] [-t] [-n] [-z] [-i] [-j] [-v] [-i] [-j]\n\n -d - cloud DB server ip address, defaulted to localhost if not specified \n -u - user name to access cloud DB, defaulted to "root" if not specified \n -p - cloud DB user password, defaulted to no password if not specified \n\n -m - the ip address of management server, defaulted to localhost if not specified\n\n -s - stop then start all running SSVMs and Console Proxies \n -r - stop then start all running Virtual Routers\n -a - stop then start all running SSVMs, Console Proxies, and Virtual Routers \n -n - restart all Guest networks \n -t - number of parallel threads used for stopping Domain Routers. Default is 10.\n -l - log file location. Default is cloud.log under current directory.\n -z - do restart only for the instances in the specific zone. If not specified, restart will apply to instances in all zones\n -v - do restart all VPCs in the entire system\n -i - do restart on instances with specific IDs. Comma-separated instance IDs to be provided. Works ony when restarting SSVMs, CPVMs or VRs\n -j - do restart on instances from the specified database table. If not specified, restart will apply on instances present in cloud.vm_instance database table. Works ony when restarting SSVMs, CPVMs or VRs\n\n" $(basename $0) >&2
+  usage_content="
+  The tool for stopping/starting running system vms and domain routers
+
+  Usage: %s: [-d] [-u] [-p] [-m] [-s] [-r] [-a] [-t] [-n] [-z] [-i] [-j] [-v] [-i] [-j]
+
+   -d - cloud DB server ip address, defaulted to localhost if not specified
+   -u - user name to access cloud DB, defaulted to \"root\" if not specified
+   -p - cloud DB user password, defaulted to no password if not specified
+   -m - the ip address of management server, defaulted to localhost if not specified
+   -s - stop then start all running SSVMs and Console Proxies
+   -r - stop then start all running Virtual Routers
+   -a - stop then start all running SSVMs, Console Proxies, and Virtual Routers
+   -n - restart all Guest networks
+   -t - number of parallel threads used for stopping Domain Routers. Default is 10
+   -l - log file location. Default is cloud.log under current directory
+   -z - do restart only for the instances in the specific zone. If not specified, restart will apply to instances in all zones
+   -v - do restart all VPCs in the entire system
+   -i - do restart on instances with specific IDs. Comma-separated instance IDs to be provided. Works ony when restarting SSVMs, CPVMs or VRs
+   -j - do restart on instances from the specified database table. If not specified, restart will apply on instances present in cloud.vm_instance database table. Works ony when restarting SSVMs, CPVMs or VRs\n\n"
```

Comment:
is there any table than vm_instance ?

Reply:
@weizhouapache the original issue requested option to provide a different VM table,

> Parametrize table name allowing cloning subsets to temporary data tables in the DB

Maybe the use-case is to copy a subset of entries in a new table and use the script on those VM entries

Reply:
@shwstppr 
ok, make sense